.github/workflows/ci.yml

```diff
@@ -0,0 +1,445 @@
+    name: "Integration / Platforms"
+    runs-on: ubuntu-latest
+
+    concurrency:
+      group: integration-platforms
+      cancel-in-progress: true
+
+    needs: integration-builds
+
+    if: github.ref == 'refs/heads/staging'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+
+          - platform: nix
+            script: |
+              build_application="$( \
+                gunzip -c ./builds/polykey-cli.closure.gz | \
+                nix-store --import | \
+                tail -1 \
+              )"
+
+              $build_application/bin/polykey
+
+          - platform: docker
+            script: |
+              docker info
+
+              mkdir $PK_TEST_TMPDIR
+
+              nix develop .#ci --command bash -c $'
+                image_and_tag="$(docker load --input ./builds/*docker* | cut -d\' \' -f3)"
+                docker tag "$image_and_tag" "polykey-cli:testtarget"
+                npm run test tests/integration/docker
+              '
+
+          - platform: linux
+            script: for f in ./builds/*-linux-*; do "$f"; done
+
+          - platform: windows
+            script: |
+              mkdir -Force "$CI_PROJECT_DIR/tmp"
+
+              Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
+
+              .\scripts\choco-install.ps1
+
+              refreshenv
+
+              npm ci --ignore-scripts
+
+              $env:Path = "$(npm root)\.bin;" + $env:Path
+
+              $env:version = $(node -p "require('./package.json').version")
+
+              npm run pkg -- `
+                --output=builds\polykey-cli-$env:version-win32-x64 `
+                --bin=dist\polykey.js `
+                --node-version=20 `
+                --platform=win32 `
+                --arch=x64
+
+              Get-ChildItem -File ./builds/*-win32-* | ForEach {& $_.FullName}
+
+          - platform: macos
+            script: |
+              version=$(node -p "require('./package.json').version")
+
+              npm ci
+
+              npm run pkg -- \
+                --output=builds/polykey-cli-$version-darwin-universal \
+                --bin=dist/polykey.js \
+                --node-version=20 \
+                --platform=darwin \
+                --arch=arm64
+
+              for f in ./builds/*-darwin-universal*; do "$f"; done
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '20'
+
+      - uses: actions/download-artifact@v2
+        with:
+          name: builds
+          path: ./builds
+
+      - name: Build
+        run: ${{ matrix.script }}
+
+      - uses: actions/upload-artifact@v4
+        with:
+          name: builds
+          path: ./builds
+
+  integration-prerelease:
+    name: "Integration / Pre-release"
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/matrixai/github-runner
+
+    concurrency:
+      group: integration-prerelease
+      cancel-in-progress: true
```

Comment:
Same here. If a new release is pushed up to staging, why should an old pre-release be cancelled?